ground/src/main/java/com/google/android/ground/persistence/local/room/converter/ConverterExt.kt

```diff
@@ -163,10 +167,10 @@ fun LocationOfInterestEntity.toModelObject(survey: Survey): LocationOfInterest =
       submissionCount = submissionCount,
       properties = properties,
       job =
-        survey.getJob(jobId = jobId)
-          ?: throw LocalDataConsistencyException(
-            "Unknown jobId ${this.jobId} in location of interest ${this.id}"
-          )
+      survey.getJob(jobId = jobId)
+        ?: throw LocalDataConsistencyException(
+          "Unknown jobId ${this.jobId} in location of interest ${this.id}"
```

Comment:
Consider returning `Result<LocationOfInterest>` and passing error there to ensure it is handled downstream.